acceptance_test.go

```diff
@@ -51,6 +52,7 @@ func TestAcceptance(t *testing.T) {
 				SourceConfig:      cfg,
 				DestinationConfig: cfg,
 				BeforeTest:        beforeTest(cfg),
+				Skip:              []string{"ResumeAtPositionCDC"},
```

Comment:
Nitpick: if you could add a comment mentioning why we're skipping this and/or a GitHub issue for that, that would be great.

Reply:
With the last changes, that was not necessary anymore. I removed it.